### ISSUE TITLE
Getting Block-scoped declaration error on run

### ISSUE THREAD

**Issue by atifazam** [egoist/vuepack#98]
Trying to run it, but getting this error:

`SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode`

Any idea?

**Comment by egoist** [comment]
yeah I guess I have to add `use strict` on the top of the file, but for now you can upgrade node.js or add it yourself.
